index.html

```diff
@@ -20,24 +20,32 @@
   <link
     href="https://fonts.googleapis.com/css2?family=Crete+Round:wght@400;500;700&family=Poppins&familty=Inter:wght@500;700&family=Roboto:wght@400;500;700&display=swap&"
     rel="stylesheet">
-  <script src='main.js'></script>
 </head>
 
 <body>
   <header class="start-screen">
-    <nav>
-      <a class="logo" href="#">Logo</a>
-      <button class="menu-btn menu">
-        <img src="assets/Icon-Menu.png" alt="menu button">
-      </button>
-      <ul class="page-links" tabindex="0">
-        <li><a href="#">Portfolio</a></li>
-        <li><a href="#">About</a></li>
-        <li><a href="#">Contact</a></li>
-        <li><a href="#"><span class="mail-icon"></span></a></li>
+    <nav id="nav">
+      <a id="logo" class="logo" href="#">Logo</a>
+      <div id="burguer" class="menu-btn">
+        <span  class="burguer"></span>
+      </div>
+      <ul id="page-links" class="page-links" tabindex="0">
+        <li id="p-l"> <a class="on-view" href="#">Portfolio</a></li>
+        <li id="a-l"> <a href="#">About</a></li>
+        <li id="c-l"> <a href="#">Contact</a></li>
+        <li id="m-l"> <a href="#"><span class="mail-icon"></span></a></li>
       </ul>
     </nav>
-    <section class="headline">
+
+    <div id="pop-menu" class="pop-menu">
+      <ul>
+        <li>Portfolio</li>
+        <li>About</li>
+        <li>Contact</li>
+      </ul>   
```

Comment:
- [ ] **Please make sure that when the user click (or tap) any option of the mobile menu, the mobile menu disappear.**
This is the basic behavior of any mobile navigation menu. Otherwise, navigation will be difficult and it will result in un-friendly UI.